src/components/multiSelect/multiSelect.js

```diff
@@ -265,6 +284,16 @@ function showMenuForSelectedItems(e) {
                                 hideSelections();
                                 dispatchNeedsRefresh();
                                 break;
+                            case 'removefromcollection':
+                                apiClient.ajax({
+                                    type: 'DELETE',
+                                    url: apiClient.getUrl('Collections/' + collectionId + '/Items', {
+                                        Ids: items.join(',')
+                                    })
+                                });
+                                dispatchNeedsRefresh();
+                                hideSelections();
```

Comment:
```suggestion
                                }).then(() => {
                                    dispatchNeedsRefresh();
                                    hideSelections();
                                });
```

Shouldn't we dispatch after the request in order for the rows to update?